schema.json

```diff
@@ -143,6 +143,15 @@
               "type": "string",
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "skills": {
```

Comment:
Naming this `"keywords"` instead of `"skills"` would be more consistent with existing use of `keywords` in other resume sections (Skills, Interests, Projects).

(Note that this PR jsonresume/resume-schema#409 is a duplicate of jsonresume/resume-schema#401, which uses `"keywords"`.)